cabot/cabotapp/migrations/0020_initial_schedule.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.conf import settings
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """Create an inital schedule with default parameters (for backwards compatibility
+           with the single-calendar version"""
+        if settings.CALENDAR_ICAL_URL:
+            schedule = orm.Schedule.objects.create(
+                name='Main',
+                ical_url=settings.CALENDAR_ICAL_URL
+            )
+            schedule.save()
```

Comment:
This is a really good idea to provide a data-migration to ensure a smooth rollout. I would recommend a bit more.
* Create a schedule with the "calendar" URL
* Iterate through all the services (`Service.objects.all()`) and add the main schedule to all services.

This ensures feature parity with what we have. Once the feature is rolled out, individual teams can make changes.